app/views/fields/_field.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1.heading-large= @field.data['field'].tr('-', ' ').humanize
```

Comment:
I don't think this should be an H1 within the overlay as there would then be multiple H1s on the page.

Reply:
Awaiting feedback from Connor before changing this but yes, I agree, it shouldn't be a H1 in the dialog.